src/d3a/models/strategy/__init__.py

```diff
@@ -108,6 +116,7 @@ def remove(self, offer):
                 return True
         except KeyError:
             self.strategy.log.warning("Could not find offer to remove")
+            return True
```

Comment:
I am trying to understand a bit more this function, if I understand correctly it returns either True in some of the branches or None (if the offer.id is not in the sold dict), shouldn't it return False to match the return value ?

Reply:
This was a quickfix for one unittest. The reason for this:
If market not in self.sold, a KeyError is raised (before it was not, because the DefaultDict just created the market in self.sold). But you are right, this should nit return True in this case. I will change (new)Line 112 to `if market in self.sold and offer.id in self.sold[market]:` to tackle this. OK?
Thanks!

Reply:
Wanted to wait for more comments first but for some reason committed and pushed it already. Sorry!!!